Fluentify.Web/Pages/Users.cshtml.cs

```diff
@@ -23,4 +26,44 @@ public async Task OnGet()
             Users = await _userService.GetAll().ToListAsync();
         }
     }
-}
\ No newline at end of file
+
+    public class LoginModel : PageModel
+    {
+        private readonly SignInManager<Users> _signInManager;
+
+        public LoginModel(SignInManager<Users> signInManager)
+        {
+            _signInManager = signInManager;
+        }
+
+        public IActionResult OnGet()
+        {
+            return Page();
+        }
+
+        public async Task<IActionResult> OnGetGoogleLoginAsync(string returnUrl = "/")
+        {
+            var redirectUrl = Url.Page("/usr_kab", pageHandler: "GoogleCallback", values: new { returnUrl });
```

Comment:
This `var redirectUrl = Url.Page("/usr_kab", pageHandler: "GoogleCallback", values: new { returnUrl });` does something weird, instead of registering usec cabinet via Google